Mathlib/Data/Matrix/RowCol.lean

```diff
@@ -26,136 +26,146 @@ variable {R : Type*} {α : Type v} {β : Type w}
 
 namespace Matrix
 
-/-- `Matrix.col u` is the column matrix whose entries are given by `u`. -/
-def col (w : m → α) : Matrix m Unit α :=
+/-- `Matrix.col ι u` is the column matrix whose entries are given by `u`.
+
+-/
+@[nolint unusedArguments]
+def col (ι : Type*) [Unique ι] (w : m → α) : Matrix m ι α :=
```

Comment:
Do you actually need `Unique` anywhere in the API?
```suggestion
def col (ι : Type*) (w : m → α) : Matrix m ι α :=
```

Reply:
updated!